File: 18-day/memory.py
from random import choice

def randomChars(chars, number):
    charsList = []
    for i in range(0, number):
        charsList.append(choice(chars))
    return charsList

characters = ['"', '#', '$', '%', '(', ')', '/', '&']
print(randomChars(characters. 4))

File: 18-day/memory.js
function randomChars(chars, number) {
  let charsList = []
  for (let i = 0; i < number; i++) {
    charsList.push(chars[Math.floor(Math.random() * chars.length)])
  }
  return charsList
}

const characters = ['"', '#', '$', '%', '(', ')', '/', '&']
console.log(randomChars(characters, 4))